src/h2olog/misc/gen_raw_tracer.py

```diff
@@ -76,6 +76,10 @@
         ("super.local.cid_set.plaintext.master_id", "master_id"),
     ]],
 
+    ["st_h2o_conn_t", [
+        ("uuid", None),
+    ]],
```

Comment:
I think we'd prefer retaining the capability of logging conn_id (of 64-bits) as well.

Reply:
`conn_id` is passed separately, but I've reverted the changes in `gen_raw_tracer.py` anyway.